playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -1,5 +1,19 @@
 ---
 debug_level: 2
+
+deployment_rhel7_ent_base:
```

Comment:
Nice! I didn't think about having to have this be outside of the deployment_vars dict, but it makes sense. Thanks for cleaning that up!